source/_integrations/intellifire.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: Intellifire 
+description: Instructions on the Intellifire Fireplace integration for Home Assistant.
+ha_category:
+  - Binary Sensor
+ha_iot_class: Local Polling
+ha_release: 0.47
+ha_codeowners:
+  - '@jeeftor'
+ha_domain: intellifire
+ha_config_flow: true
+---
+
+
+### Overview
+
+Intellifire Wifi fireplace modules provide app-based and Alexa control to various fireplace. The modules do expose an unencrypted HTTP endpoint on the network that provides status information. This integration will read that URL and create a set of sensors displaying the current fireplace state.
+
+Configuration is done via the GUI.
+
+{% include integrations/config_flow.md %}
+
+
+### Sensor Types
+
+
+The following sensors are available as either a **Binary Sensor** when dealing with on/off.
+
+### Binary Sensors
+
+- `pilot_light`: Pilot light status.
+- `fan_on`: Whether the fan is on/off.
+- `timer_on`: Whether timer-mode is engaged.
+- `thermostat_on`: Whether the thermostat is turned on.
```

Comment:
What is with the code fenced items?

Reply:
I was trying to call out the different sensors it would display .. .and so I used the code formatting on the sensor names. If that's not aligned with other documentation I'm happy to remove the fencing...

(I personally though it looked clearer ...thats all)

Reply:
Well it assume/pretends its code (the underscores add to that as well). I guess it is referring to entities object IDs or something. However, those aren't guaranteed either.

It is better to have just a normal list of human-readable sensor names and their descriptions.

Reply:
Sounds good -> I've updated it to human readable mode (and found a minor bug which I resolved)